pkg/database/neighbors_db.sql

```diff
@@ -16,8 +16,9 @@ CREATE TABLE IF NOT EXISTS users (
 CREATE TABLE userSession (
  SessionKey VARCHAR(100) PRIMARY KEY,
  UserID INT NOT NULL,
- LoginTime BIGINT not null,
- LastSeenTime BIGINT not null,
+ LoginTime BIGINT NOT NULL,
+ LastSeenTime BIGINT NOT NULL,
+ Role ENUM('NEIGHBOR', 'SAMARITAN') NOT NULL,
```

Comment:
I'd actually consider changing this to a `TINYINT(1)`.  Storage will be better (not relying on metadata) and should eventually result in better performance.  Also allows you to keep your code as the authority for what that field's value means (easier for providing migrations in the future by simply providing a version for the session table without having to change data, as an example).

Reply:
Someone said it better than me: http://komlenic.com/244/8-reasons-why-mysqls-enum-data-type-is-evil/

Reply:
I'll create another issue to address this since it's not totally related to this problem. You are absolutely correct though.